doc/src/manual/noteworthy-differences.md

```diff
@@ -107,6 +107,10 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Like many languages, Julia does not always allow operations on vectors of different lengths, unlike
     R where the vectors only need to share a common index range.  For example, `c(1, 2, 3, 4) + c(1, 2)`
     is valid R but the equivalent `[1, 2, 3, 4] + [1, 2]` will throw an error in Julia.
+  * Be aware that Julia ignores trailing commas. This can cause confusion among R users when indexing
```

Comment:
> Be aware that Julia ignores trailing commas.

Isn't this a bit too general of a statement?

 ```
julia> (1)
1

julia> (1,)
(1,)

 julia> (1,,)
 ERROR: syntax: unexpected ,
```

Reply:
Maybe add "when indexing"?

Reply:
My understanding from the discussion on discourse is that:
   - Julia will ignore ONE (and only one) trailing comma, UNLESS
   - that comma is used to signal something is a tuple

I guess I can flesh that whole nuance out? But it's not limited to indexing. e.g. 

    julia> [1, 2, 3,]
    3-element Array{Int64,1}:
     1
     2
     3

Reply:
julia> [1, 2, 3,,]
    ERROR: syntax: unexpected ,